src/main/webapp/swagger.jsp

```diff
@@ -11,15 +11,15 @@
 %>
 swagger: "2.0"
 info:
-  title: Signature Commons Data API
-  description: An API for efficient data-level queries on the Signature Commons.
+  title: <%= System.getenv("TITLE") %>
+  description: <%= System.getenv("DESCRIPTION") %>
```

Comment:
This isn't guaranteed to run in docker e.g. in development -- it would be nice to set the default here when these are empty strings or null

Reply:
won't it work already given that we defined it in the dockerfile?

Reply:
Not if we run it with `gradle tomcatWar`... which is how we use it when debugging.